content/python/concepts/built-in-functions/terms/divmod/divmod.md

```diff
@@ -0,0 +1,62 @@
+---
+Title: 'divmod()'
+Description: 'Returns the quotient and remainder of the division of two numbers.'
+Subjects:
+
+    'Computer Science'
+    'Mathematics'
+    Tags:
+    'Functions'
+    'Math'
+    'Numbers'
+    CatalogContent:
+    'learn-python-3'
+    'paths/computer-science'
+    'paths/mathematics'
+---
+
+Returns the quotient and remainder of the division of two numbers.
+
+## Syntax
+
+```py
+divmod(x, y)
+```
+
+## Example 1
+
+Use `divmod()` to calculate the quotient and remainder of dividing `10` by `3`:
+
+```codebyte/python
+result = divmod(10, 3)
+
+print(result)  # Output: (3, 1)
+```
```

Comment:
remove the comment and add an output code block (```shell). Also change the block above to ```python. There should generally be one standard example and one codebyte example.